index.html

```diff
@@ -252,14 +256,27 @@
 
     // Filter Occupations to Major Group Types
     if (options.filtered) {
-        data = data.filter(function (d) { return d.Group === options.filtered; })
+        data = data.filter(function (d) { return d.Group === options.filtered; });
     } 
 
     // Select only the occupations from the current page (slice)
     if (options.page) {
-      data = data.slice(options.page*20+1,options.page*20+21 )
+      data = data.slice(options.page*20+1,options.page*20+21 );
+      	if (data[options.page*20+22] ===null) {
```

Comment:
I wasn't able to figure out the correct condition to check if the first item of the next page does not exist. But I at least see how to get the next/previous disabled or active once I have the condition working.